src/main/java/com/example/bookstore/controller/BookController.java

```diff
@@ -50,6 +53,7 @@ public BookDto getBookById(@PathVariable
     @Operation(summary = "Find books by specific parameters ",
             description = "Returns all books that match the conditions.")
     @GetMapping("/search")
+    @PreAuthorize("hasRole('ROLE_USER') or hasRole('ROLE_ADMIN')")

```

Comment:
just remove this line to get such effect
all endpoints except for /auth and swagger require authorization per your security configuration